stack.yaml

```diff
@@ -0,0 +1,74 @@
+# This file was automatically generated by 'stack init'
```

Comment:
Hmm, is this here because of Circle CI? In that case, it would be better to have Circle CI build using Nix and cabal, since that's what encouraged for development of this project. I can make these changes myself, don't worry about it (unless you want to, in which case I'd be happy to provide any guidance) :)

Reply:
See comment below.